app/views/foreman_ansible/hosts/_tab_content.html.erb

```diff
@@ -1,4 +1,3 @@
 <div class='tab-pane' id='ansible_roles'>
-  <%= multiple_checkboxes f, :ansible_roles, @host, ansible_roles,
-    :label => _('Roles') %>
+  <%= render 'foreman_ansible/ansible_roles/select', :object => @hostgroup, :f => f %>
```

Comment:
No need to pass any `object` if you just pass `f.` and call `f.object` on the partial